examples/cosmicshear.yaml

```diff
@@ -1,17 +1,36 @@
 parameters:
   Omega_k: 0.0
-  Omega_c: 0.27
+  Omega_c: [0.25, 0.27, 0.32]  # [min, start, max]
   Omega_b: 0.045
   h: 0.67
   n_s: 0.96
-  sigma8: 0.8
+  A_s: [2.0e-9, 2.1e-9, 2.2e-9]
```

Comment:
I assume this is cosmosis speak?

Reply:
You mean A_s vs sigma8 or the three values (min,start,max)?
The latter is noted in the comment above.
The former is a DESC convention - cosmosis doesn't care.

Reply:
I meant the list of params. The DESC convention is what it is.

Reply:
I've added a comment to explain.